services/galaxytoolshed/galaxy-toolshed-downloads.service.js

```diff
@@ -0,0 +1,55 @@
+import Joi from 'joi'
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    times_downloaded: Joi.number().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedDownloads extends BaseGalaxyToolshedService {
+  static category = 'downloads'
+  static route = {
+    base: 'galaxy-toolshed/downloads',
+    pattern: ':repository/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed - Downloads',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: this.render({ downloads: 10000 }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'downloads',
+  }
+
+  static render({ downloads }) {
+    return renderDownloadsBadge({ downloads })
+  }
+
+  static transform(response) {
+    const data = this.filterRepositoryRevisionInstallInfo({
+      response,
+    })
+    return data.times_downloaded
+  }
+
+  async handle({ repository, owner }) {
+    const response = await this.fetchLastOrderedInstallableRevisionsSchema({
+      repository,
+      owner,
+      schema: repositoryRevisionInstallInfoSchema,
+    })
+    const downloads = this.constructor.transform(response)
```

Comment:
With the aforementioned changes to the schema, we're guaranteed that `response` will be an array with at least 1 element where the first element will contain the `times_downloaded` field, so we can simply just grab it (this is a large part of why we do response schema validation in the first place, as it greatly simplifies working with/processing the response object)

We can simply destructure that object, and rename the destructured value to `downloads`

```suggestion
    const { times_downloaded: downloads } = response[0]
```